proto/cometbft/abci/v1/types.proto

```diff
@@ -516,8 +516,11 @@ message Validator {
 
 // ValidatorUpdate is a singular update to a validator set.
 message ValidatorUpdate {
-  cometbft.crypto.v1.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
-  int64                        power   = 2;
+  int64  power   = 2;
+  bytes pub_key = 3;
+  string key_type = 4;
+
+  reserved 1;  // pub_key is unused in ABCI
```

Comment:
We'll need to preserve the former definition in another beta package for 0.38 a-la #2529 because this is a breaking change in `types.v1` which currently corresponds to 0.38+.

Reply:
reverted